docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,439 @@
-# Man Page
+# Man pages
+
+---
+
+### pbench-clear-results
+
+---
+
+**NAME**  
+pbench-clear-results, Clears the result directory
+
+**SYNOPSIS**  
+`pbench-clear-results [-C][--help]`
+
+**DESCRIPTION**  
+This command clears the results directory from /var/lib/pbench-agent directory
+
+**OPTIONS**  
+ -C, --config PATH  
+Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable, if defined) [required]
+
+--help  
+Show this message and exit.
+
+---
+
+### pbench-clear-tools
+
+---
+
+**NAME**  
+pbench-clear-tools, Clear registered tools by name or group
+
+**SYNOPSIS**  
+`pbench-clear-tools [-C][-n][-g][-r][--help]
+`
```

Comment:
There's a spurious newline -- lines 35 & 36 should be combined.

Ditto for most of the commands, here.